homeassistant/components/telegram_bot/webhooks.py

```diff
@@ -40,8 +63,8 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
                      last_error_date, current_status)
     else:
         _LOGGER.debug("telegram webhook Status: %s", current_status)
-    handler_url = '{0}{1}'.format(hass.config.api.base_url,
-                                  TELEGRAM_HANDLER_URL)
+
+    handler_url = "{0}{1}".format(base_url, TELEGRAM_HANDLER_URL)
```

Comment:
I think that we might have to drop the quote rules. I don't feel strongly either way or about enforcing it . Might just be a hassle instead of really a benefit?